src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonItem } from '../PersonItem/PersonItem';
+
+enum ErrorMessages {
+  SomethingWentWrong = 'Something went wrong',
+  NoPeopleOnTheServer = 'There are no people on the server',
+}
+
+export const PeopleList: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const fetchedPeople = await getPeople();
+
+        const preparedPeople = fetchedPeople.map(person => {
+          const mother = person.motherName
+            ? fetchedPeople.find(p => p.name === person.motherName) ?? null
+            : null;
+
+          const father = person.fatherName
+            ? fetchedPeople.find(p => p.name === person.fatherName) ?? null
+            : null;
+
+          return {
+            ...person,
+            mother,
+            father,
+          };
```

Comment:
- Create a separate function and move it to utils/helpers
- Avoid one-letter arguments/variable